### DISCUSSION TITLE
프론트엔드 E2E 테스트 도입

### DISCUSSION THREAD

**Discussion by Jaymyong66** [woowacourse-teams/2024-code-zap#637] [결정 사항]
반복되는 인간 QA 를 줄이기 위해 E2E 테스트를 도입을 고려해봅시다!!

**Comment by Jaymyong66** [comment]
헤인 : e2e 도입하면 좋다. cypress
월하 : e2e 도입하면 좋을거 같은데.
마위 : playwright 고공~

**Comment by Hain-tain** [comment]
playwright 도입 완료

관련 이슈
- #636 

도입 PR
- #672  (#648 와 같은 내용)
- #647
